#621: fixed ID fallback for XML merger

hohwille opened the pull request:
Fixes #621

jan-vcapgemini reviewed (commented):
LGTM. If possible it would be nice if you could also add another test for the repeated usage of merge:ids for similar elements.

hohwille commented:
> If possible it would be nice if you could also add another test for the repeated usage of merge:ids for similar elements.
I added a test for the bug I found and fixed. Your suggestion is not related to #621 
For the record: repeated usage of `merge:ids` is already covered by existing tests (e.g. `differentStrategies` or `intellijWorkspaces`).
So IMHO no further change required.
